spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,188 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :system do
+  before do
```

Comment:
- [ ] Please note that you are missing a test that checks whether you can view the user's bio. :pray: